web/modules/custom/react_tools/tools/apps/react-ppm-tool/src/App.js

```diff
@@ -154,6 +157,7 @@ class App extends Component {
     axios.post(url, options)
       .then(res => {
         let results = res.data;
+        console.log(results)
```

Comment:
@sarnikbixal Do we want to leave this here?

Reply:
No, good catch.